basiclti/basiclti-blis/src/java/org/sakaiproject/lti13/LTI13Servlet.java

```diff
@@ -117,13 +120,25 @@
 	// TODO: Rotate these after a while
 	private KeyPair tokenKeyPair = null;
 
+    protected static Ignite ignite = null;
+    protected static IgniteCache<String, String> igniteCache = null;
+
+	private static final String CACHE_NAME = LTI13Servlet.class.getName() + "_cache";
+	private static final String CACHE_PUBLIC = "key::public";
+	private static final String CACHE_PRIVATE = "key::private";
+
 	@Override
 	public void init(ServletConfig config) throws ServletException {
 		super.init(config);
 		if (ltiService == null) {
 			ltiService = (LTIService) ComponentManager.get("org.sakaiproject.lti.api.LTIService");
 		}
 
+        if (igniteCache == null) {
+            Ignite ignite = (Ignite) ComponentManager.get("org.sakaiproject.ignite.SakaiIgnite");
```

Comment:
Instead of getting the underlying Cache implementation its best to use the spring Cache Manager:

```org.sakaiproject.ignite.SakaiCacheManager```

This Cache Manager is already enabled so all you really to do in your tool is just turn on scanning with:

```<cache:annotation-driven cache-manager="org.sakaiproject.ignite.SakaiCacheManager"/>```

And use the annotations like:
@CachePut
@CacheEvict

See this guide:
https://www.baeldung.com/spring-cache-tutorial